src/runner/settings_window.cpp

```diff
@@ -153,15 +149,16 @@ BOOL run_settings_non_elevated(LPCWSTR executable_path, LPWSTR executable_args,
 
     SIZE_T size = 0;
     InitializeProcThreadAttributeList(nullptr, 1, 0, &size);
-    pptal = (PPROC_THREAD_ATTRIBUTE_LIST) new (std::nothrow) char[size];
+    auto pproc_buffer = std::make_unique<char[]>(size);
+    auto pptal = reinterpret_cast<PPROC_THREAD_ATTRIBUTE_LIST>(pproc_buffer.get());
     if (!pptal)
```

Comment:
`make_unique` throws - maybe remove `if (!pptal)` block altogether?

Reply:
Would it be possible to force it to not throw as you suggested with the previous code?

Reply:
sure, smth like `std::unique_ptr<char[]>{new(std::nothrow)...`